frontend/src/components/authentication/RegistrationForm.jsx

```diff
@@ -454,6 +709,45 @@ class RegistrationForm extends Component {
                   </label>
                 )}
               </div>
+              <div className="privacy-notice-grid" style={styles.privacyNoticeZone}>
+                <div className="privacy-notice-grid-checkbox">
+                  <input
+                    aria-invalid={!isValidPrivacyNotice && !isFirstLoad}
+                    aria-label={
+                      isValidPrivacyNotice || isFirstLoad
+                        ? LOCALIZE.authentication.createAccount.privacyNotice +
+                          LOCALIZE.authentication.createAccount.privacyNoticeLink
+                        : LOCALIZE.authentication.createAccount.privacyNoticeError +
+                          LOCALIZE.authentication.createAccount.privacyNotice +
+                          LOCALIZE.authentication.createAccount.privacyNoticeLink
+                    }
+                    id="privacy-notice-checkbox"
+                    type="checkbox"
+                    style={styles.checkbox}
+                    onChange={this.changeCheckboxStatus}
+                  />
+                </div>
+                <div className="privacy-notice-grid-description">
+                  <p>
```

Comment:
Do we need the <p> to wrap the content? It seems like it's just an extra HTML element?

Reply:
I assumed you were talking about the < p > tag.
I removed it in commit #https://github.com/code-for-canada/project-thundercat/pull/330/commits/0fa31c6f777cb4042571062857fd6fba7d44ac6e